inc/saiport.h

```diff
@@ -2108,6 +2120,233 @@ typedef sai_status_t (*sai_clear_port_pool_stats_fn)(
         _In_ uint32_t number_of_counters,
         _In_ const sai_stat_id_t *counter_ids);
 
+/**
+ * @brief List of Port Serdes attributes
+ */
+typedef enum _sai_port_serdes_attr_t
+{
+    /**
+     * @brief Start of attributes
+     */
+    SAI_PORT_SERDES_ATTR_START,
+
+    /**
+     * @brief Port ID
+     *
+     * @type sai_object_id_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @objects SAI_OBJECT_TYPE_PORT
+     */
+    SAI_PORT_SERDES_ATTR_PORD_ID = SAI_PORT_SERDES_ATTR_START,
+
+    /**
+     * @brief Port serdes control pre-emphasis
+     *
+     * List of port serdes pre-emphasis values. The values are of type sai_u32_list_t
+     * where the count is number lanes in a port and the list specifies list of values
+     * to be applied to each lane.
+     *
+     * @type sai_u32_list_t
+     * @flags CREATE_AND_SET
```

Comment:
Just thinking - Is it preferred to allow CREATE_AND_SET here as opposed to CREATE_ONLY ? I believe that for majority use cases CREATE_ONLY is sufficient, and caller can any way associate a new serdes object for a port in the event that some changes are needed at run-time. On the other hand, supporting SET requires maintain state in SAI implementations. 
I don't have any strong preference either way, but just wanted to highlight an alternate path.

Reply:
I am ok with create_only flag.  @kcudnik @tonytitus @lguohan let me know if you are fine with this one.